tests/suites/hup/suite.js

```diff
@@ -103,6 +103,29 @@ const doHUP = async (that, test, mode, hostapp, target) => {
 const initDUT = async (that, test, target) => {
 	test.comment(`Initializing DUT for HUP test`);
 
+	const bootConfig = await imagefs.interact(that.context.get().os.image.path, 1, async (_fs) => {
+		try {
+			return await require('bluebird').promisify(_fs.readFile)('/config.txt');
+		} catch (e) {
+			return undefined;
+		}
+	});
+
+	if (bootConfig) {
```

Comment:
I'd prefer to see something like `if (deviceType in ['raspberry-pi', 'rasberry-pi2', 'raspberrypi3', 'raspberrypi4-64'])` here, so this platform specific fix is applied to the correct platforms, and it's obvious that it's device type specific. Maybe it's worth considering a device type specific setup/initialization hook mechanism?

Reply:
The SV maintains a list already, though it would be better if this were in the device contract.
https://github.com/balena-os/balena-supervisor/blob/1bfcd6b791cbf42cc7e7fdfe07e61cf686e8f7cb/src/config/backends/config-txt.ts#L54-L65